cdap-proto/src/main/java/io/cdap/cdap/proto/sourcecontrol/SortOrderOptions.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.sourcecontrol;
+
+/**
+ * Sort order options for namespace and repository pipelines.
+ */
+public enum SortOrderOptions {
```

Comment:
Something like SortBy would be a better name for this, as these aren't really options, but columns to sort by.

Reply:
Though it would be good to check if we really need this. There isn't an efficient way to sort by arbitrary columns in NoSQL.

Reply:
Made the change.

Actually these are not arbitrary columns. LAST_SYNCED_DATE is indexed (LAST_MODIFIED field) and PIPELINE_NAME is a part of primary key (NAME field). To sort based on either of these 2 columns is a part of the new feature and hence, can't be neglected.

Reply:
I see. Even for an indexed field or part of the primary key, it will only be efficient in nosql for one sort order and not for the other. Do we need to have the SortOrder?

Reply:
Noted. Unfortunately, it is a requirement and needs to be there. I confirmed this with samik as well.